### DISCUSSION TITLE
Rename `babel-preset-solid` to `@babel/preset-solid`

### DISCUSSION THREAD

**Discussion by rtritto** [solidjs/solid#1544] [Ideas]
Same as:
- [@babel/preset-env](https://babeljs.io/docs/en/babel-preset-env) for compiling ES2015+ syntax
- [@babel/preset-typescript](https://babeljs.io/docs/en/babel-preset-typescript) for [TypeScript](https://www.typescriptlang.org/)
- [@babel/preset-react](https://babeljs.io/docs/en/babel-preset-react) for [React](https://reactjs.org/)
- [@babel/preset-flow](https://babeljs.io/docs/en/babel-preset-flow) for [Flow](https://flow.org/)

[Source](https://babeljs.io/docs/en/presets)

**Comment by atk** [comment]
The `@babel` namespace is reserved for official packages from the babel.js team. But maybe we can use `@solidjs/babel` instead of babel-preset-solid.

**Reply by lxsmnsyc** [reply]
and I don't think Babel would adopt the package either 🤣
